configs/template_config.yaml

```diff
@@ -146,7 +148,5 @@ layers: 2
 model_type: "retnet" # Choices: "retnet", "transformer"
 # Sequence Length (int): Context window size by number of tokens
 seq_len: 128
-# Value Embedding Dimension (int): Value embed dimension size
-value_embed_dim: 12
```

Comment:
value_embed_dim is referenced twice in models.py. If this is to be removed, it should be removed there as well. Check that does not cause further problems. I have been looking into if decoder_value_embed_dim does anything, but I am not sure, so testing on a small model to make sure it runs may help.

Reply:
It looks like it's used by the retnet, but not transformer. I will add it back in to the config, with a better default value, and remove it for the transformer code.